docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -2,8 +2,149 @@
+
+```console
+pass=$(openssl rand -base64 48 | tr -d "=+/" | head -c 32)
+echo "Password: $pass"
+echo "Password Hash: $(htpasswd -bnBC 10 "" $pass | tr -d ':\n')"
+echo "Signing Key: $(openssl rand -base64 48 | tr -d "=+/" | head -c 32)"
+```
+
+:::note
+Methods of securing the admin account are explored in greater detail [here](../40-security/10-secure-configuration.md#securing-the-admin-account).
 :::
+
+## `spec.source.helm.parameters`
+
+The most straightforward way to specify chart configuration options is by using the
+`Application`'s `spec.source.helm.parameters` field:
+
+:::info
+The parameters used are just examples, and you should use the values that are appropriate for your environment. Detailed information about available options can also be found in the [Kargo Helm Chart's README.md](https://github.com/akuity/kargo/tree/main/charts/kargo).
+:::
+
+```yaml
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kargo
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    namespace: kargo
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: ghcr.io/akuity/kargo-charts
+    chart: kargo
+    targetRevision: 1.2.0
+    helm:
+      parameters:
+        - name: api.adminAccount.passwordHash
+          # Note: A bcrypt-hashed password will contain `$` characters that
+          # MUST each be escaped as `$$`
+          value: <bcrypt-hashed password>
+        - name: api.adminAccount.tokenSigningKey
+          value: <token signing key>
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+    - CreateNamespace=true
+```
+
+
+## `spec.source.helm.values`
+
+Alternatively, instead of using `spec.source.helm`'s `parameters` field, you can use the either of the `values` or `valuesObject` fields to specify configuration options for the chart:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kargo
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    namespace: kargo
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: ghcr.io/akuity/kargo-charts
+    chart: kargo
+    targetRevision: 1.2.0
+    helm:
+      valuesObject:
+        api:
+          adminAccount:
+            passwordHash: <bcrypt-hashed password>
+            tokenSigningKey: <token signing key>
+            tokenTTL: 24h
+        controller:
+          logLevel: DEBUG
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+    - CreateNamespace=true
+```
+
+## Multi-Source Argo CD Application
+
+__Our recommended method__ is to use an `Application` with
+[multiple sources](https://argo-cd.readthedocs.io/en/stable/user-guide/multiple_sources/) to reference _both_ the Kargo Helm chart repository a `values.yaml`
+of your own from your own Git repository.
+
+__This is our recommendation because it aligns best with with GitOps principles.__
```

Comment:
```suggestion
The most advanced method covered here __is nevertheless our recommendation
because it aligns best with with GitOps principles.__ Use an `Application`
with
[multiple sources](https://argo-cd.readthedocs.io/en/stable/user-guide/multiple_sources)
to reference _both_ the Kargo Helm chart repository and a `values.yaml` file of
your own from your own Git repository.

:::info
An added benefit to this approach is that if you have other resources to
include in the Kargo installation, such as
[`SealedSecret`s](https://github.com/bitnami-labs/sealed-secrets) or
[`ExternalSecret`s](https://external-secrets.io/latest/), they also can
be obtained from your own Git repository using the second source.
:::
```